### PR TITLE
Standardize method names for 3DTiles metadata

### PR DESCRIPTION
**Pull request by jjhembd**
This PR adds `.getPropertyIds` methods to the classes [Cesium3DTileFeature](https://cesium.com/learn/cesiumjs/ref-doc/Cesium3DTileFeature.html), [Cesium3DTilePointFeature](https://cesium.com/learn/cesiumjs/ref-doc/Cesium3DTilePointFeature.html), and [ModelFeature](https://cesium.com/learn/cesiumjs/ref-doc/ModelFeature.html), for better consistency with other `3DTiles_metadata` classes. 

The existing `.getPropertyNames` methods are still present, but deprecated.

Resolves https://github.com/CesiumGS/cesium/issues/9914

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thanks for the pull request @jjhembd!
* :heavy_check_mark: Signed CLA found.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.

**Review by sanjeetsuhag — COMMENTED** [review]
Now that we're standardizing on property IDs, we should also rename the `Specs/Data/Cesium3DTiles/PointCloud/PointCloudWithUnicodePropertyNames` to use "IDs" instead of "Names". The specs that use that sample data will also need to be updated.

Otherwise, this is looking great @jjhembd. Just a couple of minor changes needed.

File: Source/Scene/ModelExperimental/ModelFeature.js [hunk 1]
```diff
@@ -154,11 +155,11 @@ ModelFeature.prototype.hasProperty = function (name) {
  *
  * @example
  * // Display all the properties for a feature in the console log.
- * const propertyNames = feature.getPropertyNames();
- * const length = propertyNames.length;
+ * const propertyNames = feature.getPropertyIds();
```
[the remark this answers is not in the document]
Comment (jjhembd) [reply Source/Scene/ModelExperimental/ModelFeature.js:158]: Thanks for catching that!

**Review by sanjeetsuhag — APPROVED** [review]
All the relevant Sandcastles work as expected and the documentation looks good. Thanks @jjhembd!

**Pull request merged**